src/query/storages/fuse/src/io/locations.rs

```diff
@@ -170,7 +172,7 @@ impl TableMetaLocationGenerator {
         let splits = loc.split('/').collect::<Vec<_>>();
         let len = splits.len();
         let prefix = splits[..len - 2].join("/");
-        let block_name = splits[len - 1];
+        let block_name = trim_vacuum2_object_prefix(splits[len - 1]);
```

Comment:
do we need to trim the 'g' in this case?

Reply:
not need